onnxruntime/core/mlas/lib/amd64/QLinearBinaryOpKernelAvx2.asm

```diff
@@ -0,0 +1,436 @@
+;++
+;
+; Copyright (c) Microsoft Corporation. All rights reserved.
+;
+; Licensed under the MIT License.
+;
+; Module Name:
+;
+;    QLinearBinaryOpKernelAvx2.asm
+;
+; Abstract:
+;
+;    This module implements the kernels for the quantized linear add
+;    for element type int8_t and uint8_t.
+;
+;    This implementation uses AVX2 instructions.
+;
+;--
+
+        .xlist
+INCLUDE mlasi.inc
+        .list
+
+        EXTERN  MLasPackBytesMM256VpshufbControl:NEAR
+        EXTERN  MLasPackBytesMM256VpermpsControl:NEAR
+
+;
+; Stack frame layout for the QLinearBinaryOp kernels.
+;
+
+QLinearBinaryOpFrame STRUCT
+        SavedXmm6 OWORD ?
```

Comment:
add spaces around struct like the other asm structs?

Reply:
Sure! Fixed.